tests/conftest.py

```diff
@@ -712,7 +712,7 @@ async def make_assertion(
         data = request_data.parameters
 
         # Check regular arguments that need defaults
-        for arg in (dkw for dkw in kwargs_need_default if dkw != "timeout"):
+        for arg in (dkw for dkw in kwargs_need_default if dkw not in ("timeout", "parse_mode")):
```

Comment:
excluding `parse_mode` is okay, but it should only happen if we're testing `send_media_group`. Please edit this line to `for arg in kwargs_need_default` (the `timeout` thingy actually is a remnant from v13) and instead add

```python
if method.__name__.endswith("_media_group"):
    # the parse_mode is applied to the first media item and we test this elsewhere
    kwargs_need_default.remove("parse_mode")
```

after the definition of `kwargs_need_default` in lines 696-700

Reply:
Done in 7b683154be006b2f591960dbf82c71fad40bc219.